Godeps/Godeps.json

```diff
@@ -1161,7 +1161,7 @@
 		},
 		{
 			"ImportPath": "github.com/docker/go-units",
-			"Comment": "v0.3.1-11-g9e638d3",
+			"Comment": "v0.3.1.12-g9e638d3",
```

Comment:
Actually, this is a mistake too

Reply:
I don't see what this has todo with your dep

Reply:
Apologies, I used `s/3.1.11/3.2.12` for this upgrade. I'm doing a full review now to make sure it didn't change anything else unintended. I'll open a issue to figure out how to make these version bumps safer/easier in the future.